api/src/main/java/eu/darkbot/api/managers/PetAPI.java

```diff
@@ -73,8 +73,10 @@ default boolean hasGear(@NotNull PetGear petGear) {
      * @param gearId gear to set when possible, null to default back to user choice
      * @throws ItemNotEquippedException if given gear is not equipped or doesn't exist
      */
+    @Deprecated
```

Comment:
again, here i'd say the module should be able to freely use this, other overwriters should not be able to
in this case i'd say the result of setGear should be a NORMAL priority instead of LOWEST, the LOWEST being the user-selected, and LOW being maybe a module that is defining a new "long-running" module via a gear selector.

Example:
 - LOWEST: user selected passive mode in bot config
 - LOW: module has its own different logic for selecting, runs on guard mode or collector
 - NORMAL: (via setGear, for a limited amount of time) set kamikaze
 - HIGH/HIGHEST: a behavior sets pet link or something else